examples/put_object.py

```diff
@@ -14,43 +14,121 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Note: YOUR-ACCESSKEYID, YOUR-SECRETACCESSKEY, my-testfile, my-bucketname and
-# my-objectname are dummy values, please replace them with original values.
-
-import os
+import io
+from datetime import datetime, timedelta
 
 from examples.progress import Progress
 from minio import Minio
-from minio.error import ResponseError
-
-client = Minio('s3.amazonaws.com',
-               access_key='YOUR-ACCESSKEYID',
-               secret_key='YOUR-SECRETACCESSKEY')
-
-# Put a file with default content-type.
-try:
-    with open('my-testfile', 'rb') as file_data:
-        file_stat = os.stat('my-testfile')
-        client.put_object('my-bucketname', 'my-objectname',
-                          file_data, file_stat.st_size)
-except ResponseError as err:
-    print(err)
-
-# Put a file with 'application/csv'
-try:
-    with open('my-testfile.csv', 'rb') as file_data:
-        file_stat = os.stat('my-testfile.csv')
-        client.put_object('my-bucketname', 'my-objectname', file_data,
-                          file_stat.st_size, content_type='application/csv')
-except ResponseError as err:
-    print(err)
-
-# Put a file with progress.
-progress = Progress()
-try:
-    with open('my-testfile', 'rb') as file_data:
-        file_stat = os.stat('my-testfile')
-        client.put_object('my-bucketname', 'my-objectname',
-                          file_data, file_stat.st_size, progress=progress)
-except ResponseError as err:
-    print(err)
+from minio.commonconfig import GOVERNANCE, Tags
+from minio.retention import Retention
+from minio.sse import SseCustomerKey, SseKMS, SseS3
+
+client = Minio(
+    "s3.amazonaws.com",
+    access_key="YOUR-ACCESSKEYID",
+    secret_key="YOUR-SECRETACCESSKEY",
+)
+
+# Upload data.
+result = client.put_object(
+    "my-bucket", "my-object", io.StringIO("hello"), 5,
```

Comment:
It looks like this needs to be io.BytesIO(b"hello")

Reply:
Done